userspace/libscap/scap_procs.c

```diff
@@ -557,6 +557,11 @@ static int32_t scap_proc_add_from_proc(scap_t* handle, uint32_t tid, int parentt
 	}
 
 	//
+	// Gathers the exepath
+	//
+	snprintf(tinfo->exepath, target_res, "%s", filename);
```

Comment:
From the [https://linux.die.net/man/2/readlink](readlink) documentation 

> On success, readlink() returns the number of bytes placed in buf. On error, -1 is returned and errno is set to indicate the error.

So I used the return value for the size, I can change it to `sizeof(tinfo->exepath)` if you think that it is better.